tests/interfaces/test_qnode_torch.py

```diff
@@ -81,6 +88,9 @@ def circuit(a):
 
     def test_execution_with_interface(self, dev_name, diff_method):
         """Test execution works with the interface"""
+        if diff_method == "backprop":

```

Comment:
How come this case doesn't support backprop?

Reply:
Because this test explicitly tests the interface, but backprop requires/has no interface :)